### DISCUSSION TITLE
Problem with inputs value

### DISCUSSION THREAD

**Discussion by reziamini** [livewire/livewire#3405] [7. Bugs (archived)]
### Discussed in https://github.com/livewire/livewire/discussions/3354

<div type='discussions-op-text'>

<sup>Originally posted by **rezaamini-ir** July 17, 2021</sup>
There is a problem when we are trying to update an input.
Some of the input character values will be deleted when we update the value of others' input.
Take a look at this video : 

https://user-images.githubusercontent.com/29504334/126029127-80988558-e764-4f47-9992-e507a75b8290.mp4

We had 6 characters in the second input and when we updated the next value (Which was a checkbox) 2 character of the second input was deleted.
I think it is an issue with pulling new data, and It appears when we have an internet speed issue.
Because this problem appears in the production and with local developing, we can't feel that.

I suppose that minimizing the size of pulled data will fix it.</div>

**Comment by denizozturk** [comment]
Share these component codes (blade and class)

**Comment by reziamini** [comment]
@danharrin We have this problem with all of the components and it's not about just one single component!

**Comment by denizozturk** [comment]
> @danharrin We have this problem with all of the components and it's not about just one single component!

So can you share the code of any blade file?
